src/test/java/io/confluent/examples/streams/IntegrationTestUtils.java

```diff
@@ -277,4 +279,87 @@ private static boolean continueConsuming(final int messagesConsumed, final int m
         30000,
         "Expected values not found in WindowStore"); }
 
+  static <K, V> Map<K, V> drainTableOutput(final String outputTopic,
```

Comment:
Can we add a JavaDoc comment here? This method is super useful but maybe not clear what it does if one does not look into the implementation.

Reply:
You bet. Good idea.

Reply:
I added docs for the corresponding `produce` method as well. Thanks for pointing this out.